CONTRIBUTING.md

```diff
@@ -135,26 +215,18 @@ Once all of the above steps are met, the following process will be followed:
   - The owner will then respond to the email sent as part of the pull request, informing the group of the completion of the request.
 - If the request does not meet any of the requirements, the pull request will not be merged, and the necessary fixes for acceptance will be communicated back to the partner team.
 
-### Adding Tests
-
-Regarding the test project, one thing that's important is to name the test project by adding a ".Tests" suffix to the folder name for the folder containing your project. For example, the test project for "Compute\Management.Compute" should be named 'Compute.Tests'
-
-  - This is for improving CI performance so to find exactly one copy of your test assembly.
-  - Also, due to test dependencies, the test project should build both .NET 4.5.2 and NETStandard 1.4. For example, check out "src\SDKs\Resource\Resource.tests"
-
 ### Client Library Tested OSs and .NET Versions
 
-|                        | Linux (Ubuntu 16.04) | MacOS 10.13 | Windows Server 2016 |
-|------------------------|----------------------|-------------|---------------------|
-| **.NET Core 2.1**  |  x                   |      x      |          x          |
-
+|                   | Linux (Ubuntu 16.04) | MacOS 10.13 | Windows Server 2016 |
+| ----------------- | -------------------- | ----------- | ------------------- |
+| **.NET Core 2.1** | x                    | x           | x                   |
 
 ### Issues with Generated Code
 
-Much of the SDK code is generated from metadata specs about the REST APIs. Do not submit PRs that modify generated code. Instead, 
-  - File an issue describing the problem,
-  - Refer to the the [AutoRest project](https://github.com/azure/autorest) to view and modify the generator, or
-  - Add additional methods, properties, and overloads to the SDK by adding classes in the 'Customizations' folder of a project
+Much of the SDK code is generated from metadata specs about the REST APIs. Do not submit PRs that modify generated code. Instead,
 
+- File an issue describing the problem,
+- Refer to the the [AutoRest project](https://github.com/azure/autorest) to view and modify the generator, or
+- Add additional methods, properties, and overloads to the SDK by adding classes in the 'Customizations' folder of a project
 
 ![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2FREADME.png)
```

Comment:
Looks like this is in the wrong file. It should be in the README.md not the Contributing.md doc.